modules/web/src/app/node-data/basic/provider/anexia/template.html

```diff
@@ -30,18 +30,42 @@
                      required>
   </km-number-stepper>
 
-  <km-number-stepper [formControlName]="Controls.DiskSize"
-                     mode="errors"
-                     label="Disk Size in GB"
-                     min="1"
-                     required>
-  </km-number-stepper>
+  <ng-container formArrayName="disks">
+    <ng-container *ngFor="let diskForm of disks.controls; let i = index">
+      <div [formGroup]="diskForm"
+           fxLayout="column"
+           fxLayout="row"
+           fxLayoutGap="10px">
+
+        <km-number-stepper formControlName="size"
+                           mode="errors"
+                           label="Disk Size in GB"
+                           min="0"
+                           fxFlex="45">
+        </km-number-stepper>
+
+        <km-autocomplete label="Disk Type"
+                         fxFlex="40"
+                         formControlName="performanceType"
+                         [isLoading]="isLoadingDiskTypes"
+                         [options]="diskTypes">
+        </km-autocomplete>
```

Comment:
Just to confirm, is it allowed to enter custom `performanceType` value? i.e. a value which is not inside the available disk types.

Reply:
Configuring a disk using a performanceType which is not included in the available disk types will fail during VM provisioning in the machine-controller. (Similarly when configuring non-existing VLANs)